sqlite.go

```diff
@@ -508,6 +518,7 @@ func (s *stmt) ExecContext(ctx context.Context, args []driver.NamedValue) (drive
 		return nil, err
 	}
 	_ = row // TODO: return error if exec on query which returns rows?
+	<-done
```

Comment:
maybe do this conditionally on it being non-nil to avoid the alloc+close of it in the no-context-key case?

Reply:
SGTM, done.